filters/bevel/src/BevelFilter.ts

```diff
@@ -101,11 +101,11 @@ class BevelFilter extends Filter
      */
     get lightColor(): number
     {
-        return utils.rgb2hex(this.uniforms.lightColor);
+        return new Color(this.uniforms.lightColor).toNumber();
```

Comment:
Many places you're using `new Color()`, for simple conversions, consider using `Color.shared.setValue()` instance instead as this does not generate extra garbage.